plugins/org.jboss.tools.locus.mockito/1.9.5/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,101 @@
+eclipse.preferences.version=1
```

Comment:
Well, then +1 to these changes project-only changes, but NOT to the creation of a new Activator since that would be a fork of the original jar. Need feedback from Max, Alexey, Mickael, etc.